lib/storage/table.go

```diff
@@ -465,7 +473,14 @@ func (tb *table) finalDedupWatcher() {
 			ptw.pt.isDedupScheduled.Store(false)
 		}
 	}
-	d := timeutil.AddJitterToDuration(time.Hour)
+
+	// adds 25% jitter in order to prevent thundering herd problem at cluster version
```

Comment:
```suggestion
	// adds 25% jitter in order to prevent thundering herd problem
	// https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7880
```